applications/luci-app-socatg/Makefile

```diff
@@ -0,0 +1,14 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=luci-app-socatg
+PKG_VERSION=1.2
+PKG_RELEASE:=1
+PKG_LICENSE:=MIT
+PKG_MAINTAINER:=drophair <https://github.com/big-tooth>
+LUCI_TITLE:=LuCI Support for socatg
+LUCI_PKGARCH:=all
+LUCI_DEPENDS:=+socat
+# PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)
+
+include $(INCLUDE_DIR)/package.mk
```

Comment:
Please also replace the this lines with:

```
include ../../luci.mk

# call BuildPackage - OpenWrt buildroot signature
```
Have a look at the other application Makefiles in the repository